packages/typescript-estree/package.json

```diff
@@ -64,6 +64,9 @@
     "jest-specific-snapshot": "*",
     "make-dir": "*",
     "tmp": "^0.2.1",
+    "typescript": "^4.3.2"
+  },
+  "peerDependencies": {
```

Comment:
We have an implicit peer dependency via the `peerDependenciesMeta` property.

The reason we don't explicitly declare the peer dependency is because on older versions of yarn and npm, they do not understand `peerDependenciesMeta`.
So if we add an explicit peer dep - then they will log warnings at install time.

In the past this annoyed many users 😅 that had our packages installed (but not used) transitively.

TL;DR - remove the explicit peer dep